packages/dashboard-provider/lib/types.ts

```diff
@@ -0,0 +1,19 @@
+export interface DashboardProviderOptions {
```

Comment:
Could we move this interface to be inside `./lib/DashboardProvider.ts`? No reason to move the useful documentation info further away from the implementation ;)